src/docs/install.md

```diff
@@ -17,47 +17,42 @@ After you have pants installed, you'll need to
 Recommended Installation
 ------------------------
 
-To set up pants in your repo, we recommend installing our self-contained `pants` bash script
-in the root (ie, "buildroot") of your repo:
+To set up Pants in your repo, we recommend installing our self-contained `pants` bash script
+in the root (i.e. the "buildroot") of your repo:
 
     :::bash
-    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants && touch pants.ini
+    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants
 
-The first time you run the new `./pants` script it will install the latest version of pants (using
-virtualenv) and then run it.  It's recommended though, that you pin the version of pants.  To do
-this, first find out the version of pants you just installed:
+Start by running the below command to auto-generate a `pants.ini` config file with sensible defaults.
 
     :::bash
-    ./pants -V
-    1.0.0
+    ./pants generate-pants-ini
 
-Then add an entry like so to `pants.ini` with that version:
+Running `./pants` for the first time will install the latest version of Pants, using virtualenv and first trying to use Python 3.6 and then falling back to Python 2.7. (Note that Pants 1.16.0 will be the last version to support Python 2.7.)
 
-    :::ini
-    [GLOBAL]
-    pants_version: 1.0.0
+This command also pins the `pants_version`. When you'd like to upgrade Pants, just edit the version in `pants.ini` and `./pants` will self-update on the next run.
 
-When you'd like to upgrade pants, just edit the version in `pants.ini` and pants will self-update on
-the next run.  This script stores the various pants versions you use centrally in
-`~/.cache/pants/setup`.  When you switch back and forth between branches pants will select the
-correct version from your local cache and use that.
+The script stores the various virtual environments you use centrally in
```

Comment:
This paragraph also seems like TMI.

Reply:
I agree with all these comments. In this last update, I removed all references to how the `./pants` script works, i.e. to virtualenv and the default Python strategy. I don't think any of that explanation was necessary.

Also it's a bit repetitive with the recent changes, as we log where the virtualenv was created: https://github.com/pantsbuild/setup/blob/f19663d9d7985b4c5d60415c5fc50b9a46544b05/pants#L136.

Let me know if I removed too much.